src/utils.ts

```diff
@@ -13,3 +13,5 @@ export const processRegExpPattern = (pattern: string) => {
 
   return new RegExp(source || pattern, flags);
 };
+
+export const normalize = (text: string ) => (text || '').toUpperCase();
```

Comment:
Where it is used? If you moved the function to utils, there should be also an import statement changes somewhere

Reply:
The import is in this file `https://github.com/IvanFon/super-labeler-action/blob/develop/src/conditions/isOpen.ts`.

I target `develop` branch because noticed that @IvanFon merge my changes with `isOpen` condition on `develop`. I saw that there was some changes in my original PR and `normalize` function meant to be imported. However it was not placed in `utils` (probably some over-missed)